inception-project/inception-reporting-dashboard
# Feature 22 support reading special inception project tags

**What's in the PR**
* Implemented #22
* Hardcoded translations for code used in GeMTeX
* Also reorganzied functions

**How to test manually**
1. Create an INCEpTION project.
2. Add a code from one of these lists [1,2] to the description
3. Import the project into the Dashboard
4. The exported project status JSON object should contain the translated codes

**Automatic testing**

**Documentation**
